setup.py

```diff
@@ -491,6 +491,33 @@ def check_cudnn_version_and_warn(global_option: str, required_cudnn_version: int
         )
     )
 
+    # CUDA group norm V2 is tested on SM100
+    if bare_metal_version >= Version("12.7"):
+        arch_flags = ["-gencode=arch=compute_100,code=sm_100"]
```

Comment:
nit-picking: I'd use `> Version("12.7")` or `>= Version("12.8")` 
Question: no need to add sm120?

another nit-picking: if `TORHC_CUDA_ARCH_LIST` is correctly set, e.g., `TORCH_CUDA_ARCH_LIST="9.0 10.0 10.0a 12.0 12.0a", then I don't think it necessary to modify define this variable, IIUC.

Reply:
I think the condition check should be consistent with https://github.com/NVIDIA/apex/pull/1889/files . Maybe both two pull requests should change 12.7 to 12.8 .

Currently, the extension is only intended to be run on 148 SMs. The reason is that template arguments depend on SM_COUNT, and for future GPU cards whose SM_COUNT is unknown, GroupNorm v2 is not guaranteed to be runnable or to be faster than v1.
As for utilizing compile-time unknown SM_COUNT, I would like to develop the feature in a new pull request in the future.

Thus only `sm_100` is used, no matter which `TORCH_CUDA_ARCH_LIST` is.